sh/validate_mysql_database_access.sh

```diff
@@ -10,6 +10,9 @@ MYSQL_DB_NAME=$(grep 'mySqlDbName' $config_file | awk -F'=' '{ print $2 }')"_cor
 if [[ $type =~ ensembl ]]; then
     MYSQL_USER=ensro
 elif [[ $type =~ wbps ]]; then
+    echo "===========================================================================" >&2
+    echo "WARNING: WBPS references are now fetched from Ensembl Genomes please check." >&2
+    echo "===========================================================================" >&2
```

Comment:
Maybe change "please check" for a more concrete action.

Reply:
Fixed in [3b3f535](https://github.com/ebi-gene-expression-group/atlas-annotations/pull/57/commits/3b3f53556116f6f2771382fba1cfda4eb2f385ad)